src/Hangashore/lib/drivers/wireless.ts

```diff
@@ -15,8 +19,15 @@ const makeWirelessDriver = (name: string) => {
             surge: data.surge,
             yaw: data.yaw,
         }));
+        const rssi$ = new Subject<number>();
 
-        motion$.subscribe(motion => m.send(Buffer.from(motion)));
+        motion$.subscribe(motion => {
+            m.send(Buffer.from(motion));
+            m.recv().then(response => rssi$.onNext(response.readInt16BE(response.length - 2)));
```

Comment:
While this works, this PR should wait until the underlying microcontrollers API has been updated for two reasons:

1. The new implementation will be less awful (see LakeMaps/microcontrollers#2)
2. This here knows too much about how the messages are encoded (i.e. the fact that the RSSI is the last two bytes)—this should be simply `response.rssi`